src/applications/ChildBenefitsClaim/StartPage.tsx

```diff
@@ -75,8 +77,14 @@ const StartPage: React.FC<{onStart: React.MouseEventHandler, onBack: any}> = ({o
           Start now
         </Button>
         <p className='govuk-body'>
-          If you have any questions about Child Benefit, <a
-            href='https://www.gov.uk/government/organisations/hm-revenue-customs/contact/child-benefit' className='govuk-link'>contact the Child Benefit helpline</a>.
+          If you have any questions about Child Benefit,{' '}
```

Comment:
{' '}?

Reply:
this would be fixed in another translation story